README.md

```diff
@@ -40,6 +40,11 @@ keyring provider present, so you can use the auxiliary credential helper to
 minimize Microsoft SSO prompts during a development session. The order is
 important since you want the `cache` credential helper checked before `msal`.
 
+**NOTE:** This does not store the cache related to the OAuth2.0 refresh token
+that is stored in the keyring. This cache is useful for not needing to
+re-authenticate for 90-days. Have a look at the Device Authorization Grant
```

Comment:
Is it always 90 days or is that configurable in the Azure tenant? `s/90-days/90 days/` otherwise.

Reply:
It's not configurable by the Azure tenant or tenant owner. However, the expiration could be sooner if certain account actions occur to invalidate it sooner.

* https://learn.microsoft.com/en-us/entra/identity-platform/refresh-tokens#token-lifetime
* https://learn.microsoft.com/en-us/entra/identity-platform/refresh-tokens#token-revocation

Reply:
I should probably provide these links in the README........

Reply:
Thanks, that makes sense. Maybe just "for up to 90 days" then?

Reply:
Done :). I included the links as well for any user that might be interested.